accepted/README.md

```diff
@@ -0,0 +1,3 @@
+# Accepted
+
+When a proposal has been `accepted`, it will be scheduled for work and upon completion be moved to `implemented`.
```

Comment:
What do you think of linking back to the overall proposal process doc? For example:

```suggestion
When a proposal has been `accepted`, it will be scheduled for work and upon completion be moved to `implemented`. For more information, see the [NuGet proposal process](https://github.com/NuGet/Home/tree/proposals-cleanup/meta#nuget-proposal-process).
```

Reply:
@loic-sharma Can you do this for each README & I'll merge?

Reply:
Actually, nevermind. I want to merge now as promised, so I'll do it.